### PR TITLE
ANA-436: Swaption lifecycle

### PR DESCRIPTION
**Pull request by zvyagin1**
# Pull Request Checklist

- [ ] Read the [contributing guidelines](../blob/master/docs/CONTRIBUTING.md)
- [ ] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwaptionExamples.cs [hunk 1]
```diff
@@ -132,5 +133,113 @@ public void InterestRateSwaptionInlineValuationExample(ModelSelection.ModelEnum
...
+        /// of PV (under CTVoM model) and in particular, equals to the payoff
+        ///
+        /// Cash-settled case: works the same as the others. This means, we get the cashflow
+        /// and upsert that back into LUSID.
+        ///
+        /// Physically-settled case: Cashflow would be paying the strike to obtain the underlying.
+        /// There is additional code to get the underlying in the GetValuation call as well as then
+        /// upserting the underlying back into the portfolio. 
+        /// </summary>
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney,"Cash")]
```
Comment (rizwansaeed) [inline sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwaptionExamples.cs:149]: this needs a unique LusidFeature (https://gitlab.com/finbourne/salesengineering/lusid-features/-/blob/master/features.yml#L365) most likely a F22-x

Comment (jz-finbourne) [reply sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwaptionExamples.cs:149]: @azvyagin1, I can add the above key for you (I'm adding some on my side currently).

Comment (zvyagin1) [reply sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwaptionExamples.cs:149]: thank you!

Comment (jz-finbourne) [reply sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwaptionExamples.cs:149]: It's "F22-35" :)

Comment (jz-finbourne) [reply sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwaptionExamples.cs:149]: from https://gitlab.com/finbourne/salesengineering/lusid-features/-/commit/651bcad4257edda4e7c23a4f4148a65ff4fbc6ed

Comment (jz-finbourne) [reply sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwaptionExamples.cs:149]: Need a rebase / resolve conflict btw

**Pull request merged**